app/models/designation.js

```diff
@@ -61,6 +62,58 @@ export default Model.extend({
     return this.get("state").capitalize();
   }),
 
+  stateIcon: Ember.computed('stateIcon', function () {
+    const state = this.get("state");
+    switch (state) {
+      case "awaiting_dispatch":
+      case "scheduled":
+        return "clock-o";
+      case "processing":
+        return "list";
+      case "submitted":
+        return "envelope";
+      case "dispatching":
+        return "paper-plane";
+      case "cancelled":
+        return "thumbs-down";
+      case "closed":
+        return "lock";
+      default:
+        return "";
+    }
+  }),
+
+  transportIcon: Ember.computed("transportIcon", function() {
+    const key = this.get("transportKey");
+    switch (key) {
+      case "gogovan_transport":
+        return "truck";
+      case "collection_transport":
+        return "male";
+      default:
+        return "";
+    }
+  }),
+
+  transportLabel: Ember.computed("transportLabel", function() {
+    const key = this.get('transportKey');
+    return this.get("i18n").t(`order_transports.${key}`);
+  }),
+
+  transportKey: Ember.computed("transportKey", function() {
+    const transportType = this.get('orderTransport.transportType');
+    if (transportType) {
```

Comment:
I don't think we need this if condition here.

Reply:
It was just for structure, as the else block would theoretically handle transports for stockit orders. I can remove it, and re-add it once(if) we move to stockit orders